catalog.html

```diff
@@ -0,0 +1,253 @@
+          </a>
+          <span class="product-card-type">Гостиница</span>
+          <span class="product-card-price">От 5000 ₽</span>
+          <a class="product-card-link" href="#">Подробнее</a>
+          <a class="favourites-link" href="#">В избранное</a>
+          <div class="rating">Рейтинг: 9,2</div>
+        </li>
+
+        <li class="product-card">
+          <a class="product-card-link" href="#">
+            <img class="product-card-image" src="#" alt="Горы на фоне ясного">
+            <h3 class="product-card-title">Desert Quail Inn</h3>
+          </a>
+          <span class="product-card-type">Гостиница</span>
+          <span class="product-card-price">От 2500 ₽</span>
+          <a class="product-card-link" href="#">Подробнее</a>
+          <a class="favourites-link" href="#">В избранное</a>
+          <div class="rating">Рейтинг: 6,9</div>
+        </li>
+
+        <li class="product-card">
+          <a class="product-card-link" href="#">
+            <img class="product-card-image" src="#" alt="Горы на фоне ночного звёздного неба">
+            <h3 class="product-card-title">GreenTree Inn</h3>
+          </a>
+          <span class="product-card-type">Гостиница</span>
+          <span class="product-card-price">От 1500 ₽</span>
+          <a class="product-card-link" href="#">Подробнее</a>
+          <a class="favourites-link" href="#">В избранное</a>
+          <div class="rating">Рейтинг: 5,0</div>
+        </li>
+      </ul>
+
+      <ul class="pagination">
+        <li class="pagination-item">
+          <a class="pagination-link pagination-current">1</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">2</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">3</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">4</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">...</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">10</a>
+        </li>
+      </ul>
+      
+      <label class="product-on-page">
+        Гостиниц на странице
+        <select name="product-on-page-count">
+          <option value="4">4</option>
+          <option value="4">5</option>
+          <option value="4">6</option>
+        </select>
+      </label>
+    </section>
+  </main>
+
+  <footer class="page-footer">
+    <section class="newsletter">
+      <h2 class="newsletter-title">Подпишитесь на рассылку</h2>
+      <p class="newsletter-desc">Только полезная информация и никакого спама, честное бойскаутское!</p>
+      <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+        <input class="field" type="email" name="newsletter-email" placeholder="Ваш e-mail">
+        <button class="button newsletter-button" type="button">Подписаться</button>
```

Comment:
```suggestion
        <button class="button newsletter-button" type="submit">Подписаться</button>
```
Так как эта кнопка формы отвечает за отправку данных.